libs/sdk-ui-kit/src/List/InsightListItemDate.tsx

```diff
@@ -36,3 +43,51 @@ export const InsightListItemDate: React.FC<IInsightListItemDateProps> = ({ confi
 
     return <FormattedDate value={config.date} format="shortWithYear" />;
 };
+
+const META_TIMEZONE = "Europe/Prague";
+
+const FORMAT_DATE = "yyyy-MM-dd";
+const FORMAT_YEAR = "yyyy";
+
+const sameFormatted = (firstDate: Date, secondDate: Date, formatter: string) =>
+    format(firstDate, formatter) === format(secondDate, formatter);
+
+/**
+ * @internal
+ */
+export interface IDateTimeConfigOptions {
+    dateTimezone?: string;
+    now?: Date;
+}
+
+/**
+ * Build date time config for InsightListItemDate component.
+ *
+ * @param date
+ * @param options
+ * @returns date time config
+ *
+ * @internal
+ */
+export function getDateTimeConfig(
+    date: string,
+    options: IDateTimeConfigOptions = {},
+): IInsightListItemDateConfig {
+    const { dateTimezone = META_TIMEZONE, now = new Date() } = options;
+
+    const dateWithTimezoneInLocal = parseISO(date);
+    const dateWithTimezone = utcToZonedTime(dateWithTimezoneInLocal, dateTimezone);
+
+    const yesterday = subDays(now, 1);
+
+    const isToday = sameFormatted(dateWithTimezoneInLocal, now, FORMAT_DATE);
+    const isYesterday = sameFormatted(dateWithTimezoneInLocal, yesterday, FORMAT_DATE);
+    const isCurrentYear = sameFormatted(dateWithTimezoneInLocal, now, FORMAT_YEAR);
```

Comment:
These format based comparisons are quite weird, can we use the comparison functions directly?
* https://date-fns.org/v2.19.0/docs/isSameDay
* https://date-fns.org/v2.19.0/docs/isSameYear

Reply:
Fixed. Good suggestion 👌